### ISSUE TITLE
tensorflow-hub problems

### ISSUE THREAD

**Issue by hosea7456** [tensorflow/models#9517]
**Hi,
I want to use the tensorflow-hub to extract features from a pre-trained model. And my code is:**

hub_path = '/home/simclr/pre_models/r50_1x_sk1/hub/'
module = hub.Module(hub_path, trainable=False)
outputs = module(inputs=X_test, signature="default", as_dict=False)
print(type(outputs))
print(outputs.shape)
sess = tf.compat.v1.InteractiveSession()
out_np = sess.run(outputs)

**###output**
<class 'tensorflow.python.framework.ops.Tensor'>
(10000, 2048)

I want to use the sess.run function  to convert the tensor outputs to the numpy one, but I failed, the errors is:

Original stack trace for 'module_apply_default/base_model/batch_normalization_54/batchnorm/ReadVariableOp_2':
  File "feture.py", line 65, in <module>
    outputs = module(inputs=X_test, signature="default", as_dict=False)
  File "/usr/local/lib/python3.6/dist-packages/tensorflow_hub/module.py", line 269, in __call__
    name=name)
  File "/usr/local/lib/python3.6/dist-packages/tensorflow_hub/native_module.py", line 618, in create_apply_graph
    import_scope=relative_scope_name)
  File "/usr/local/lib/python3.6/dist-packages/tensorflow/python/training/saver.py", line 1462, in import_meta_graph
    **kwargs)[0]
  File "/usr/local/lib/python3.6/dist-packages/tensorflow/python/training/saver.py", line 1486, in _import_meta_graph_with_return_elements
    **kwargs))
  File "/usr/local/lib/python3.6/dist-packages/tensorflow/python/framework/meta_graph.py", line 799, in import_scoped_meta_graph_with_return_elements
    return_elements=return_elements)
  File "/usr/local/lib/python3.6/dist-packages/tensorflow/python/util/deprecation.py", line 507, in new_func
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/tensorflow/python/framework/importer.py", line 405, in import_graph_def
    producer_op_list=producer_op_list)
  File "/usr/local/lib/python3.6/dist-packages/tensorflow/python/framework/importer.py", line 513, in _import_graph_def_internal
    _ProcessNewOps(graph)
  File "/usr/local/lib/python3.6/dist-packages/tensorflow/python/framework/importer.py", line 243, in _ProcessNewOps
    for new_op in graph._add_new_tf_operations(compute_devices=False):  # pylint: disable=protected-access
  File "/usr/local/lib/python3.6/dist-packages/tensorflow/python/framework/ops.py", line 3624, in _add_new_tf_operations
    for c_op in c_api_util.new_tf_operations(self)
  File "/usr/local/lib/python3.6/dist-packages/tensorflow/python/framework/ops.py", line 3624, in <listcomp>
    for c_op in c_api_util.new_tf_operations(self)
  File "/usr/local/lib/python3.6/dist-packages/tensorflow/python/framework/ops.py", line 3510, in _create_op_from_tf_operation
    ret = Operation(c_op, self)
  File "/usr/local/lib/python3.6/dist-packages/tensorflow/python/framework/ops.py", line 1949, in __init__
    self._traceback = tf_stack.extract_stack()

**How can I solve this problem? Or maybe the method which I use the tensorflow-hub to extract features was wrong.**

**Comment by saikumarchalla** [comment]
@hosea7456  Seems the issue is more related to TF HUB repo. Please post your query [here](https://github.com/tensorflow/hub/issues/new). Thanks!

**Comment by hosea7456** [comment]
@saikumarchalla  Got it, thank you!
